File: JS/task_2.js
export const task2 = 'Задача 2. Написать программу, которая должна найти и вывести повторяющийся символ в ' +
    'слове «Hello». UPD Поиск всех повторяющихся символов в слове'

export function findDuplicateChar(word) {
    const charCount = {};
    for (const char of word) {
        if (charCount.hasOwnProperty(char)) {
            charCount[char] += 1;
        } else {
            charCount[char] = 1;
        }
    }
    return charCount;
}

File: python310/task_2.py
def find_duplicate_char(word):
    char_count = {}

    for char in word:
        if char in char_count:
            char_count[char] += 1
        else:
            char_count[char] = 1

    return char_count


word = input("Введите слово: ")
result = find_duplicate_char(word)
if result:
    print(f"В слове '{word}': ")
    for char, count in result.items():
        if count > 1:
            print(f"символ {char}: повторы {count}")
else:
    print(f"В слове '{word}' нет повторяющихся символов")